### ISSUE TITLE
Change display of drop down

### ISSUE THREAD

**Issue by khemebuen** [leovo2708/ngx-treeview#93]
Is it possible to change the display of the DropdownTreeviewComponent?  For our project we want to only show a folder icon and when they click on the icon we show the tree.  If the component used a template like the tree items do then we could change the the display to not only show text but show an icon or icon and text

**Comment by leovo2708** [comment]
Yes, you can see code to implement DropdownTreeviewComponent, it will reuse TreeviewComponent, so you will not have to write many codes.

**Comment by khemebuen** [comment]
I am using the drop down select and have changed the nga-dropdown-treeview component with a custom drop down component which just shows a folder but the drop down directive is not working as my tree is not hiding

**Comment by khemebuen** [comment]
Still trying to reuse the TreeviewComponent to only how a folder icon and when the folder icon is selected the tree view shows.  I setup my html for my control like this but it does not work.  It just shows the whole tree and button separate.  Am I missing smething:

`<div class="dropdown" ngxDropdown>
    <button class="btn" [ngClass]="buttonClass" type="button" role="button" ngxDropdownToggle>      
        <span class="fa fa-folder"></span>
    </button>
    <div ngxDropdownMenu aria-labelledby="dropdownMenu" (click)="$event.stopPropagation()">
        <div class="dropdown-container">
            <ngx-treeview [config]="config" [headerTemplate]="headerTemplate" [items]="items" [itemTemplate]="itemTemplate" (selectedChange)="onSelectedChange($event)">
            </ngx-treeview>
        </div>
    </div>
</div>`

**Comment by SteffiKeranJ** [comment]
How to initialize the init of all as false?

**Comment by SSDSynergyAsad** [comment]
![image](https://user-images.githubusercontent.com/101694623/160452307-19bfdf2f-856c-4ef5-aed4-8aee4d3f6fee.png)
![image](https://user-images.githubusercontent.com/101694623/160456666-76f6f00e-841d-4e63-a686-39cb1c88ebab.png)
![image](https://user-images.githubusercontent.com/101694623/160454024-8164c775-195e-4987-97d2-dbc3dfe210a6.png)
![image](https://user-images.githubusercontent.com/101694623/160455009-4ec7a4dc-4de1-49d2-9bc5-de4e14d8883c.png)

how to reuse placeholder Value actually i create three time dropdown but every dorpdown placeholder value change so how to reuse...
